spec/shr_oncologycore.txt

```diff
@@ -42,151 +42,167 @@ Concept:        LNC#21858-6 "Grade Cancer"
-        includes  0..1 TNMPathologicRegionalNodesClassification
-        includes  0..1 TNMPathologicDistantMetastasesClassification
-
-Element:        StageTiming
+    0..0        FindingMethod // The expectation is that each member of the panel will declares its own method; if added here, it would be redundant.
+
+
+EntryElement:   TNMClinicalStagePanel
+Based on:       CancerStagePanel
+Concept:        SCT#385349001   "Clinical stage (observable entity)"
+Description:    "Clinical staging is performed prior to surgery, based on evidence such as physical examination, imaging, and/or biopsy."
+                FindingTopicCode is SCT#385349001 "Clinical stage (observable entity)"
+                PanelMembers.Observation
+                    includes  0..1 TNMClinicalStageGroup
+                    includes  0..1 TNMClinicalPrimaryTumorClassification
+                    includes  0..1 TNMClinicalRegionalNodesClassification
+                    includes  0..1 TNMClinicalDistantMetastasesClassification
+
+EntryElement:   TNMPathologicStagePanel
+Based on:       CancerStagePanel
+Concept:        SCT#405978005   "Pathologic stage (observable entity)"
+Description:    "Pathologic stage is based on examination of tissue samples removed during surgery, in addition to physical examination and imaging."
+                FindingTopicCode is SCT#405978005   "Pathologic stage (observable entity)"
+                PanelMembers.Observation
+                    includes  0..1 TNMPathologicStageGroup
+                    includes  0..1 TNMPathologicPrimaryTumorClassification
+                    includes  0..1 TNMPathologicRegionalNodesClassification
+                    includes  0..1 TNMPathologicDistantMetastasesClassification
+
+
+Element:        CancerStageTiming 
 Concept:        SCT#260869008 "Timing of stage"
 Description:	"Indicates when the staging was done, in terms of treatment landmarks."
 Value:          CodeableConcept from StageTimingPrefixVS
 
-Element:        StageGroup
-// did not use precoordinated codes because only available for clinical and pathologic.
-Based on:       CodedNonIndependentFinding
+
+Abstract Element: TNMStageGroup
+Based on:       CodedNonLaboratoryObservation
 Concept:        MTH#C1302362 "Stage Grouping"
 Description:    "The classification resulting from cancer staging. Each type of cancer has its own Each type of cancer has its own stage groups based on where the cancer has grown and spread. There are either four or five stage groups per cancer. (Source: NCCN Cancer Staging Guide)"
                 FindingResult from http://loinc.org/vs/LL240-3
-                FindingTopicCode is MTH#C1302362 "Stage Grouping" 
+                FindingTopicCode is MTH#C1302362 "Stage Grouping"
+0..1            FindingMethod from TNMTumorStagingSystemVS if covered
+                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+1..1            CancerStageTiming
```

Comment:
CancerStageTiming might not be used with all staging systems. Suggest making 0..1

Reply:
+1